04-files.md

```diff
@@ -16,8 +16,8 @@ The only thing that's missing is a library with a rather unpleasant name:
 import glob
 ~~~
 
-The `glob` library contains a single function, also called `glob`,
-that finds files whose names match a pattern.
+The `glob` library contains a function, also called `glob`,
+that finds files and directories whose names match a pattern.
```

Comment:
To me this makes it sound like it will return directories in the list as well (like ls) rather than files and their paths. Can you come up with a clearer way to say this?